File: BE/routes/evaluation.routes.js
const express = require('express');
const router = express.Router();
const evaluationController = require('../controllers/evaluation.controller');
const { authenticateToken } = require('../middleware/auth');
const { requireRole } = require('../middleware/role');

router.put('/:submissionId', authenticateToken, requireRole('company'), evaluationController.evaluateSubmission);

module.exports = router;

File: BE-Python/routes/evaluation_routes.py
from flask import Blueprint
from middleware.auth import authenticate_token
from middleware.role import require_role
from controllers import evaluation_controller

evaluation_bp = Blueprint('evaluation', __name__)

@evaluation_bp.route('/<int:submission_id>', methods=['PUT'])
@authenticate_token
@require_role('company')
def evaluate_submission(submission_id):
    return evaluation_controller.evaluate_submission(submission_id)
